platform/pulp/platform/models/generic.py

```diff
@@ -86,6 +86,20 @@ def items(self):
         # iterable, rather than making it __getitem__ for each key
         return ((kv.key, kv.value) for kv in self.manager.all())
 
+    def get(self, key, default=None):
+        """
```

Comment:
This provides a nice convenience.  I found this very helpful when writing the `howto.py` and dealing with notes and scratchpad.